pyiron_base/jobs/job/generic.py

```diff
@@ -690,9 +694,9 @@ def run(
                 if repair and self.job_id and not self.status.finished:
                     self._run_if_repair()
                 elif status == "initialized":
-                    self._run_if_new(debug=debug)
+                    return self._run_if_new(debug=debug)
```

Comment:
Per the other discussion and your comment, I think I lean towards your alternative idea of holding the resulting `Future` object somewhere else (e.g. the `Server`) rather than returning it. Just so that this run mode looks more like the others.

Reply:
Done